### PR TITLE
feat: show a11y features on public site

### LINKED ISSUE #975: Detroit: Show accessibility features on public
**Acceptance Criteria**
* When I have added accessibility features to the listing from Partners Portal, then I expect to see them on the listing detail page and the preview 
* When I have added accessibility features (any), then I should see the Accessible badge on the listing photo both on the detail page and on the browse page 
* When I filter by the accessibility features, they should filter by the accessibility features present on the listing 

**Notes**
* Use the new accessibility features, not the unit ones (mobility, hearing, visual) for detail display, badge and filters
* If none, do not show section
* I think the filtering work should be done, just verifying that it's all working 

**Designs** 
Detail 
https://www.figma.com/file/D2fNPS5Ucll8uz7hyMUQGB/bloom-public-single-listing?node-id=1158%3A15157

Browse
https://www.figma.com/file/ymKHAWFxakuVvc1vNDAmnm/bloom-public-browse?node-id=829%3A10294

### PR DESCRIPTION
**Pull request by emilyjablonski**
## Issue

- Closes #975 

## Description

(1) Show accessibility features in a bulleted list on the public site under the features category
(2) Show a tag on the image card that the listing is accessible if it has at least one accessibility feature

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/addresses technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Add and remove a11y features from a listing in partners and ensure the bullet points and tags come through

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes
- [ ] I have exported any new pieces in ui-components

### CODE REVIEW

**Comment by emilyjablonski** [comment]
Waiting on a design question before finishing the tag feature

**Comment by emilyjablonski** [comment]
Blocked by: https://github.com/bloom-housing/bloom/pull/2531

**Comment by emilyjablonski** [comment]
The long term fix is to move the tags so that if there were multiple tags on the image card they would display properly. Current plan for speed is to merge this as-is with the acknowledgement that there are not any community types right now so at most there is going to be one tag (accessible) showing on a listing card.

File: sites/public/src/ListingView.tsx [hunk 1]
```diff
@@ -235,6 +239,17 @@ export const ListingView = (props: ListingProps) => {
     </>
   )
 
+  const additionalInformationCard = (cardTitle: string, cardData: string) => {
```
Comment (emilyjablonski) [inline sites/public/src/ListingView.tsx:242]: We can pull this DRY into core

Comment (seanmalbert) [reply sites/public/src/ListingView.tsx:242]: Created an issue in Bloom off of this comment. https://github.com/CityOfDetroit/bloom/issues/1026

**Review by seanmalbert — APPROVED** [review]
lgtm!

**Pull request merged**